.github/workflows/knowlg-service.yml

```
@@ -92,7 +94,21 @@ jobs:
           echo "Pushed Docker image: $REGISTRY_URL/${IMAGE_NAME}:${IMAGE_TAG}"
+          echo "image=$REGISTRY_URL/${IMAGE_NAME}:${IMAGE_TAG}" >> $GITHUB_OUTPUT
+
+  cve-scan:
+    needs: build-and-push
+    if: needs.build-and-push.outputs.image != ''
+    permissions:
+      contents: read
+      packages: read
+      security-events: write
+      actions: read
+    uses: ./.github/workflows/cve-check.yml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/cve-check.yml": open /home/jailuser/git/.github/workflows/cve-check.yml: no such file or directory

(workflow-call)